src/utils/hints/namesHints.js

```diff
@@ -11,4 +11,16 @@ export const namesHints = {
   expired: 'Block height and estimated date and time when the Name was detached from the address.',
   lastBuyer: 'Address of the last account which paid for the Name and the price of the Name.',
   lastOwner: 'Last owner of the Name.',
+  bidder: 'Account of the highest bidder.',
+  bid: 'Price of the highest bid.',
+  ownedSince: 'Block height and estimated date and time when the account became the owner of the Name.',
+  status: 'Current state of the Name in the blockchain.',
+  customPointers: 'User-defined pointers that point to any valid value.',
```

Comment:
`That point to any valid value` sounds too generic to me. What about `User-defined pointers with custom keys.`?

Reply:
Sure thing, sounds better :+1: I pulled the previous value straight from the feature definition in one of the github issues